Formula/global.rb

```diff
@@ -35,17 +37,20 @@ class Global < Formula
   skip_clean "lib/gtags"
 
   resource "Pygments" do
-    url "https://files.pythonhosted.org/packages/cb/9f/27d4844ac5bf158a33900dbad7985951e2910397998e85712da03ce125f0/Pygments-2.5.2.tar.gz"
-    sha256 "98c8aa5a9f778fcd1026a17361ddaf7330d1b7c62ae97c3bb0ae73e0b9b6b0fe"
+    url "https://files.pythonhosted.org/packages/ba/6e/7a7c13c21d8a4a7f82ccbfe257a045890d4dbf18c023f985f565f97393e3/Pygments-2.9.0.tar.gz"
+    sha256 "a18f47b506a429f6f4b9df81bb02beab9ca21d0a5fee38ed15aef65f0545519f"
   end
 
   def install
     system "sh", "reconf.sh" if build.head?
 
     xy = Language::Python.major_minor_version "python3"
     ENV.prepend_create_path "PYTHONPATH", libexec/"lib/python#{xy}/site-packages"
-    pygments_args = %W[build install --prefix=#{libexec}]
-    resource("Pygments").stage { system "python3", "setup.py", *pygments_args }
+
+    python3 = Formula["python@#{xy}"].opt_bin/"python3"
+    resource("Pygments").stage do
+      system python3, *Language::Python.setup_install_args(libexec)
+    end
```

Comment:
```suggestion
    resource("Pygments").stage do
      system "python3", *Language::Python.setup_install_args(libexec)
    end
```
`xy` comes from querying the `python3` found in `PATH` so we may as well just use the one there directly.